gcp-vertex-ai-mojo-scorer/README.md

```diff
@@ -0,0 +1,112 @@
+# DAI Deployment Template for GCP Vertex AI
+
+The docker image that is built by this project can be pushed to gcr.io and used for scoring
+Driverless AI Mojos in GCP Vertex AI. 
+
+## Building
+
+Make sure you are in the working directory `dai-deployment-templates`. Typing `pwd` in the terminal
+shell should have a similar output to `/my/path/to/dai-deployment-templates`.
+
+You can build the Docker image in two ways. The first only includes the ability to score and the second
+includes the ability to set a preprocessing script.
+
+* Run the following command to build without preprocessing script option: 
+  ```shell script
+  ./gradlew build -PdockerRepositoryPrefix=gcr.io/your/repository -PdockerUsePython=false
+  ```
+  and the docker image required for GCP Vertex AI will be in the directory `gcp-vertex-ai-mojo-scorer/build`:
+  ```shell script
+  /path/to/dai-deployment-templates/gcp-vertex-ai-mojo-scorer/build/jib-image.tar
+  ```
+
+* Run the following command to build with preprocessing script option:
+```shell script
+./gradlew build -PdockerRepositoryPrefix=gcr.io/your/repository
+```
+and the docker image required for GCP Vertex AI will be in the directory `gcp-vertex-ai-mojo-scorer/build`:
+```shell script
+/path/to/dai-deployment-templates/gcp-vertex-ai-mojo-scorer/build/jib-image.tar
+```
+
+* Load the resulting `jib-image.tar` file to docker
+```shell script
+docker load < /path/to/dai-deployment-templates/gcp-vertex-ai-mojo-scorer/build/jib-image.tar
+``` 
+
+* Follow the steps explained here in Google Documentation: https://cloud.google.com/run/docs/building/containers, to 
+push the image to gcr.io.
+
+## Deploying
+
+To deploy the container follow the steps in Google Documentation here to import the model:
+https://cloud.google.com/vertex-ai/docs/general/import-model
+
+There is one requirement for the container. You __MUST__ include the following environment variables:
+* MOJO_GCS_PATH = `gs://path/to/pipeline.mojo`
+* LICENSE_GCS_PATH = `gs://path/to/driverless/ai/license.sig`
+
+If you built the Docker image with the preprocessing script option, you also __MUST__ include the following environment variable:
```

Comment:
**MUST**? I would be careful with this. Even if there is the option to use preprocessing stage, it shouldn't be required.